cmd/infracost/run.go

```diff
@@ -154,9 +154,6 @@ func runMain(cmd *cobra.Command, cfg *config.Config) error {
 	case "diff":
 		b, err = output.ToDiff(r, opts)
 		out = fmt.Sprintf("\n%s", string(b))
-	case "table_deprecated":
```

Comment:
We will need to remove the [root_deprecated](https://github.com/infracost/infracost/blob/master/cmd/infracost/root_deprecated.go) command as well since it uses this format. Happy for that to be done in a separate PR though. What do you think?